app/src/main/java/com/concordia/mcga/helperClasses/DatabaseHelper.java

```diff
@@ -29,50 +29,50 @@
      */
     private DatabaseHelper(Context context) {
         super(context, DATABASE_NAME, null, 1);
```

Comment:
private methods IMO don't need javadoc as they are not exposed to be used by others